tsconfig.json

```diff
@@ -18,13 +18,7 @@
         "gulpfile.ts", // tslint gets quite angry if we don't do this
         "src",
         "tasks",
-        // "test", // Uses its own tsconfig.json until all nullability issues are fixed.
+        "test",
```

Comment:
Does this mean the tests will packaged into the distributed extension?

Reply:
I didn't see any test files included in the extension.js dist folder, so I think we're good there.